extensions/wikia/CreateNewWiki/js/WikiBuilder.js

```diff
@@ -300,14 +304,22 @@ define('ext.createNewWiki.builder', ['ext.createNewWiki.helper', 'wikia.tracker'
 		checkDomain();
 		var selected = $(this).val();
 
-		if (selected && selected !== NO_SUBDOMAIN_LANGUAGE) {
-			wikiDomainCountry.html(selected + '.');
+		if (shouldCreateLanguageWikisWithPath) {
+			if (selected && selected !== NO_SUBDOMAIN_LANGUAGE) {
+				wikiBaseDomain.text(wikiaBaseDomain + '/' + selected);
+			} else {
+				wikiBaseDomain.text(wikiaBaseDomain);
+			}
 		} else {
-			wikiDomainCountry.html('');
-		}
+			if (selected && selected !== NO_SUBDOMAIN_LANGUAGE) {
+				wikiDomainCountry.html(selected + '.');
```

Comment:
This should be `.text()` as a best practice since `selected` was retrieved from the DOM using `.val()`, so it is plain text and should be put back as plain text.